packages/react-composites/src/composites/CallComposite/types/VideoRenderingControlOptions.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Customization options video rendering.
+ *
+ * @beta
+ */
+export type VideoRenderingControlOptions = {
+  /**
+   * Show a loading spinner over a remote video when video data is loading
+   * @defaultValue true
+   */
+  remoteVideoStreamLoadingSpinner: boolean;
```

Comment:
We want to try avoid booleans as they can't be extended well. Consider
```suggestion
  videoStreamLoadingOverlay: 'none' | 'spinner';
```

Reply:
We have decided to just remove videoRenderingControlOptions